.travis.yml

```diff
@@ -64,6 +65,9 @@ jobs:
 
     - name: Coverage
       rust: nightly
+      before_install:
+        # TODO vk: Not sure why this is needed but without it I get some errors related to gyp.
+        - rm -rf /home/travis/.cache/node-gyp
```

Comment:
see this added here and removed immediately again in #888. Is this a temporary crutch?